### PR TITLE
MOSIP-27352 script changes

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: db_upgrade_scripts/mosip_audit/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql [hunk 1]
```diff
@@ -8,4 +8,7 @@
 -- Modified Date        Modified By         Comments / Remarks
 -- -------------------------------------------------------------------------------------------------
 
-\c mosip_audit sysadmin
\ No newline at end of file
+\c mosip_audit sysadmin
+
+--This only need to execute if no rows added after upgradescripts for host_ip column---------------
+ALTER TABLE audit.app_audit_log ALTER COLUMN host_ip TYPE varchar(16) USING host_ip::varchar;
```
Comment (vishwa-vyom) [inline db_upgrade_scripts/mosip_audit/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql:14]: If IPv6 is used, length might be high. Should we increase this to 50 rather ?

File: db_upgrade_scripts/mosip_audit/upgrade.properties [hunk 2]
```diff
@@ -7,6 +8,5 @@ SYS_ADMIN_USER=
 SYS_ADMIN_PWD=
 DEFAULT_DB_NAME=postgres
 DBUSER_PWD=
-CURRENT_VERSION=
-TARGET_VERSION=
-ACTION=
\ No newline at end of file
+CURRENT_VERSION=1.1.5.5
+UPGRADE_VERSION=1.2.0.1-B1
```
Comment (vishwa-vyom) [inline db_upgrade_scripts/mosip_audit/upgrade.properties:12]: this is a common file, should we prefix the versions ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: db_upgrade_scripts/mosip_audit/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql (hunk 1 above)
Comment (sowmya695) [reply db_upgrade_scripts/mosip_audit/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql]: updated

File: db_upgrade_scripts/mosip_audit/upgrade.properties (hunk 2 above)
Comment (sowmya695) [reply db_upgrade_scripts/mosip_audit/upgrade.properties]: removed

**Review by vishwa-vyom — APPROVED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**